Services/CO.CDP.DataSharing.WebApi.Tests/UseCase/GenerateShareCodeUseCaseTest.cs

```diff
@@ -5,53 +5,130 @@
 using CO.CDP.OrganisationInformation.Persistence.Forms;
 using FluentAssertions;
 using Moq;
+using CO.CDP.Authentication;
```

Comment:
Consider using `EntityFactory.GivenOrganisation` to reduce repetitive setup code and improve test maintainability

Reply:
done.